src/core/raster/qgsrasterlayertemporalproperties.cpp

```diff
@@ -77,7 +74,7 @@ bool QgsRasterLayerTemporalProperties::readXml( const QDomElement &element, cons
 
 QDomElement QgsRasterLayerTemporalProperties::writeXml( QDomElement &element, QDomDocument &document, const QgsReadWriteContext &context )
 {
-  Q_UNUSED( context )
+  Q_UNUSED( context );
```

Comment:
```suggestion
  Q_UNUSED( context )
```
otherwise it raises a warning in QtCreator